tests/Lens/CompoundLensTest.js

```diff
@@ -204,7 +204,7 @@ describe('CompoundLens', () => {
       ).toEqual(
         {
           cToken: cEth._address,
-          underlyingPrice: "1000000000000000000",
+          underlyingPrice: "0",
```

Comment:
Can we add a test that sets the underlying price of Eth so we can verify it goes non-zero after that happens?